src/sickle.ts

```diff
@@ -20,6 +20,20 @@ export function formatDiagnostics(diags: ts.Diagnostic[]): string {
       .join('\n');
 }
 
+// Return true if a class declaration has a superclass (as declared with
```

Comment:
Nit: use `/** jsdoc */`, prefer 3rd person voice for method comments (`Returns ...`)